docs/tracingpolicy/README.md

```diff
@@ -0,0 +1,1193 @@
+# Tracing Policy
+
+`TracingPolicy` is a user-configurable Kubernetes custom resource (CR) that
+allows users to trace arbitrary events in the kernel and optionally define
+actions to take on a match, for enforcement for example. Currently, two types
+of events are supported: kprobes and tracepoints, but others may be added in
+the future (e.g., uprobes) by following a similar approach.
+
+Note that `TracingPolicy` can be considered low-level since they require
+knowledge about the Linux kernel and containers to be written correctly.
```

Comment:
to be written correctly. In the future, we are considering to add a high-level RuntimeSecurityPolicy which would take this abstraction away.

Reply:
added